lib/cocoapods/installer/xcode/pods_project_generator/aggregate_target_installer.rb

```diff
@@ -195,13 +155,6 @@ def create_acknowledgements
             end
           end
 
-          # @return [Pathname] the path of the bridge support file relative to the
-          #         sandbox.
-          #
-          # @return [Nil] if no bridge support file was generated.
-          #
-          attr_reader :bridge_support_file
```

Comment:
No longer needed in this file since the support was moved to `aggregate_target.rb`